recipes/prismatic/meta.yaml

```
@@ -0,0 +1,167 @@
+{% set version = "1.2.1" %}
+{% set build = 0 %}
+
+{% set processor = processor or 'cpu' %}
+
+package:
+  name: prismatic_split
+  version: {{ version }}
+
+source:
+  url: https://github.com/prism-em/prismatic/archive/{{ version }}.tar.gz
+  sha256: 844f659575dd98999b7c4864e850a7ec735f297ab1fb6c2d2be91d7f9720fe70
+  patches:
+    - patch_apple_target_lib.txt
+
+build:
+  number: {{ build }}
+  string: "{{processor}}_h{{ PKG_HASH }}_{{ build }}"
+
+outputs:
+  - name: prismatic
+    build:
+      number: {{ build }}
+      skip: True  # [win]
+      string: "{{processor}}_h{{ PKG_HASH }}_{{ build }}"
+    requirements:
+      run:
+        - {{ pin_subpackage('prismatic_cli', max_pin='x.x') }}
```

I feel like I already suggested this but I might be mixing up with another PR. If so, I'm sorry and just point me to the comment.

Should these be 
```suggestion
        - {{ pin_subpackage('prismatic_cli', exact=True) }}
```
Personally I would find it unintuitive if `conda install prismatic=1.2.3` installs `1.2.10` (made up version numbers of course). People can always install the components individually if that's really what they want.

---

You might have suggested it already - but it is a sensible suggestion which bears making twice if you did. I'll apply this change now.